Doc/howto/unicode.rst

```diff
@@ -451,6 +392,91 @@ other".  See
 list of category codes.
 
 
+Lowercasing Strings
+-------------------
+
+The string :meth:`~str.casefold` method converts a string to lowercase
+following an algorithm described by the Unicode Standard.
+Case-folding rules can be complicated: for example, the German letter
+'ß' (code point U+00DF) becomes the pair of
+lowercase letters 'ss'.
+
+::
+
+    >>> street = 'Gürzenichstraße'
+    >>> street.casefold()
+    'gürzenichstrasse'
+
+You might be accustomed to using the :meth:`~str.lower` method for
+this, but :meth:`~str.lower` uses a set of locale-specific conversions
```

Comment:
str.lower doesn't depend on your configured locale.

Reply:
Resolved as part of the rewriting of the casefold() discussion.